configs/workflow.yml

```
@@ -34,3 +34,9 @@ vivaldi:
   bookmarkOpenId: "05008a94-3fd7-44d4-aa10-b0f2a495d360"
   profileListId: "093cb3e1-761a-4f34-83f6-36507c8c6eff"
   profileSetId: "0d4fbc34-60ed-4223-98c9-5a93a103aa5c"
+
+brave:
+  bookmarkListId: "aaa9ecf5-d269-4757-b555-61b2d6dee497"
+  bookmarkOpenId: "a5008a94-3fd7-44d4-aa10-b0f2a495d360"
+  profileListId: "a93cb3e1-761a-4f34-83f6-36507c8c6eff"
+  profileSetId: "ad4fbc34-60ed-4223-98c9-5a93a103aa5c"
```

Those GUIDs should be generated (without manually adding `a`):

```suggestion
  bookmarkListId: "aaa9ecf5-d269-4757-b555-61b2d6dee497"
  bookmarkOpenId: "a5008a94-3fd7-44d4-aa10-b0f2a495d360"
  profileListId: "a93cb3e1-761a-4f34-83f6-36507c8c6eff"
  profileSetId: "ad4fbc34-60ed-4223-98c9-5a93a103aa5c"
```

---

I don't get the problem with that. Therefore I am not sure how to create them. Would this from https://www.famkruithof.net/uuid/uuidgen be ok?

```
f7cb33ad-15d9-4fa9-b723-524f5ad191a0
681d644f-4ce9-4d26-8846-dd1a35eda744
74df3946-baf6-402b-ac8f-9bad696a617a
fa89a6db-ef1b-4b50-96fb-957750e391ef
```